kagenti/installer/app/components/registry.py

```diff
@@ -43,13 +44,19 @@ def install():
             registry_ip = service.spec.cluster_ip
 
             container = f"{config.CLUSTER_NAME}-control-plane"
+            container_engine = ContainerEngine(config.CONTAINER_ENGINE)
+
             subprocess.run(
-                ["docker", "exec", container, "sh", "-c",
+                [container_engine, "exec", container, "sh", "-c",
```

Comment:
ContainerEngine is an enum object (`container_engine = ContainerEngine(config.CONTAINER_ENGINE)`), but here it is used as a string - it would be better to use `str(container_engine)` or `container_engine.value`